t/03-testapi.t

```diff
@@ -587,21 +587,38 @@ subtest 'validate_script_output' => sub {
     throws_ok {
         validate_script_output('script', sub { m/error/ })
     } qr/output not validating/, 'Die on output not match';
+    throws_ok {
+        validate_script_output('script', qr/error/)
+    } qr/output not validating/, 'Die on output not match';
```

Comment:
please use new, unique test descriptions so that the output is not repeated in console output and the specific test locations can be identified in case of failures, e.g.

```suggestion
    } qr/output not validating/, 'Die on output not match for regex';
```

Reply:
ok, commented after merge. I would appreciate if you follow-up with these changes in another PR.

Reply:
done. 😄 
https://github.com/os-autoinst/os-autoinst/pull/2179